ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+# Payment Processor Comparison
+
+Most new merchants will likely only consider the price of the service. Since BTCPay is free, that may have led you here and if so, welcome. 
+
+We want users to understand not only BTCPay, but also how payments can be processed on behalf of a merchant. In order to help users understand trade-offs when using various cryptocurrency payment processing methods. We don't know which payment processors provide which services. That will need to be part of your own research. The following list is a good place to start:
+
+* [Features](#Features)
+* [Security](#Security)
+* [Privacy](#Privacy)
+* [Decentralized](#Decentralized)
+* [Fiat](#Fiat)
+
+----
+
+## Features
+
+Every payment processor has features, here are some BTCPay features:
+
+* **Bitcoin** - Accepting Bitcoin is the first step.
+* **Altcoins** - Accept cryptocurrency alternatives to Bitcoin.
+* **Lightning** - Rapid Bitcoin microtransactions using the Lightning Network.
+* **Integrations** - Connect to your website using WooCommerce, Drupal, Magneto, Prestashop and custom integrations.
+* **Point Of Sale** - POS Interfaces for physical stores.
```

Comment:
Include payment button?

Reply:
@bitcoinshirt When you say payment button, are you talking about how merchants will have a "pay with Bitcoin" button shown on their website/POS? Meaning it adds another payment method, right?

Reply:
By Payment button, I mean the payment button from the store settings which can be embedded into HTML. It's like an additional app, like PoS. https://store.demo.btcpayserver.org/?page_id=44
So it's just another app, hence the feature, mostly used for donations, but merchants (like CasaHodl for example, use it their e-commerce solutions as well).